### ISSUE TITLE
Dashboard not building

### ISSUE THREAD

**Issue by HoloPanio** [HoloPanio/duxcore#68]
<img width="973" alt="image" src="https://user-images.githubusercontent.com/30759238/165781229-44f7bd9b-cc44-4259-949d-678facee821a.png">

Can somebody address this issue and fix it, it's causing builds to fail, also needs to be done on the core-instances branch

**Comment by HoloPanio** [comment]
> looks like a dependency conflict. have you tried updating the hcaptcha and the react package?

I have tried updating both and neither update has fixed it

**Comment by TheOtterlord** [comment]
it worked before right? could it be a breaking change was published in a minor release? I'll try this in a bit with the version specified in the package.json file

**Comment by HoloPanio** [comment]
> it worked before right? could it be a breaking change was published in a minor release? I'll try this in a bit with the version specified in the package.json file

It only began occurred after I recompiled our yarn.lock file.

**Comment by HoloPanio** [comment]
It looks like the version did break, it is still errored in VSCode, and now we have another build problem.

**Comment by HoloPanio** [comment]
```
info  - Using webpack 5. Reason: no custom webpack configuration in next.config.js https://nextjs.org/docs/messages/webpack5
info  - Checking validity of types  
info  - Creating an optimized production build .node:internal/crypto/hash:67
  this[kHandle] = new _Hash(algorithm, xofLen);
                  ^

Error: error:0308010C:digital envelope routines::unsupported
    at new Hash (node:internal/crypto/hash:67:19)
    at Object.createHash (node:crypto:130:10)
    at BulkUpdateDecorator.hashFactory (/Users/holopanio/projects/duxcore/website/node_modules/next/dist/compiled/webpack/bundle5.js:130015:18)
    at BulkUpdateDecorator.update (/Users/holopanio/projects/duxcore/website/node_modules/next/dist/compiled/webpack/bundle5.js:129917:50)
    at /Users/holopanio/projects/duxcore/website/node_modules/next/dist/compiled/webpack/bundle5.js:53479:9
    at /Users/holopanio/projects/duxcore/website/node_modules/next/dist/compiled/webpack/bundle5.js:26311:16
    at FSReqCallback.readFileAfterClose [as oncomplete] (node:internal/fs/read_file_context:68:3) {
  opensslErrorStack: [ 'error:03000086:digital envelope routines::initialization error' ],
  library: 'digital envelope routines',
  reason: 'unsupported',
  code: 'ERR_OSSL_EVP_UNSUPPORTED'
}
```

**Comment by TheOtterlord** [comment]
@HoloPanio https://stackoverflow.com/questions/69692842/error-message-error0308010cdigital-envelope-routinesunsupported

**Comment by TheOtterlord** [comment]
The workarounds don't seem great...

**Comment by HoloPanio** [comment]
After looking at that I remember this error very clearly now, I will resolve this and push changes to see if it is fixed, but can you look and see if there is a bug on either react/next about this?

**Comment by TheOtterlord** [comment]
There are a few in next, and they seem to lead to a webpack issue: https://github.com/webpack/webpack/issues/14532

**Comment by HoloPanio** [comment]
[hCaptcha/react-hcapthca#127](https://github.com/hCaptcha/react-hcaptcha/issues/127)

**Comment by TheOtterlord** [comment]
Fixed in 309be2f0efe237e0ed6fab856ae3d07b347e0487
